guardian/cdk-cli
# Update template package.json file

## What does this change?

This PR updates the template `package.json` to use to most recent version of the available package.

## How to test

Run the `init` command and then install the deps in the new repo. Check that everything still works as expected.

## How can we measure success?

New projects created via the `init` command get the latest dependencies.
